Emby.Server.Implementations/Udp/UdpServer.cs

```diff
@@ -1,112 +1,47 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
+using System.Net.Sockets;
 using System.Text;
+using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
 using MediaBrowser.Controller;
 using MediaBrowser.Model.ApiClient;
-using MediaBrowser.Model.Events;
-using MediaBrowser.Model.Net;
-using MediaBrowser.Model.Serialization;
 using Microsoft.Extensions.Logging;
 
 namespace Emby.Server.Implementations.Udp
 {
     /// <summary>
-    /// Provides a Udp Server
+    /// Provides a Udp Server.
     /// </summary>
-    public class UdpServer : IDisposable
+    public sealed class UdpServer : IDisposable
     {
         /// <summary>
         /// The _logger
         /// </summary>
         private readonly ILogger _logger;
-
-        private bool _isDisposed;
-
-        private readonly List<Tuple<string, bool, Func<string, IPEndPoint, Encoding, CancellationToken, Task>>> _responders = new List<Tuple<string, bool, Func<string, IPEndPoint, Encoding, CancellationToken, Task>>>();
-
         private readonly IServerApplicationHost _appHost;
-        private readonly IJsonSerializer _json;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="UdpServer" /> class.
+        /// The _udp client.
```

Comment:
Is this necessary? And if you comment one of the fields, you should comment them all 😄

Reply:
Do you mean this specific comment, or comments in general? This is actually why I am wary of requiring documentation. It alsmost always leads to comments like these.

Reply:
This is an old comment, I didn’t write it

Reply:
The diff didn't show that